### DISCUSSION TITLE
Picture inspiration for a new species

### DISCUSSION THREAD

**Discussion by ZachZent** [tsunamayo/Starship-EVO#4474] [Suggestions]
Something I found online. Special forces, but they totally look like evil Jawas. Maybe the "hood" can be the articulated part like the eyeball alien's ears
![image](https://user-images.githubusercontent.com/36980446/149647107-75e30cb7-2e3b-48aa-96a0-b43477d39e92.png)

**Comment by tsunamayo** [comment]
Lol indeed I can feel the dark jawa vibe...
